rules/eero.com.xml

```diff
@@ -0,0 +1,12 @@
+<sitename name="eero.com">
+  <docname name="Terms of Service">
+    <url name="https://eero.com/legal/tos" xpath="//div[@class='legal-content']" lang="EN">
+      <norecurse name="arbitrary"/>
+    </url>
+  </docname>
+  <docname name="Privacy policy for eero Devices, Applications and Services">
+    <url name="https://eero.com/legal/privacy" xpath="//div[@class='legal-content']" lang="EN">
```

Comment:
These should be `//div[contains(@class, 'legal-content')]`

Reply:
So  do you want to change that or should we merge it as it is now, with `//div[@class='legal-content']`?

Reply:
Ah right, you did that in c72b6c33d5a - great!